templates/web/zurich/header.html

```diff
@@ -86,6 +86,9 @@
 
                 <div class="content[% " $mainclass" | html IF mainclass %]" role="main">
                 [% IF NOT admin %]
+                  [% IF bodyclass.match('frontpage') %]
```

Comment:
Do we want to add a and date <= 2021-07-12 style statement here so that it automatically disappears after the end date?

Reply:
That is a smart idea, will do.